### PR TITLE
Create an Author class

### PR DESCRIPTION
**Pull request by ksigei**
**In this milestone, I was able to:**
- Create an Author class with an association to the Item class (in a separate .rb file).
- I was able to define and add the following properties in the constructor method:

          [ ]   id - int
          [ ]   first_name - string
          [ ]   last_name - string
          [ ]   items - array

-I also added a fucntionality to add new items which:

          [ ]  Takes an instance of the Item class as an input
          [ ]  Adds the input item to the collection of items
          [ ]  Adds self as a property of the item object (by using the correct setter from the item object)

### CODE REVIEW

**Review by bigmosi — APPROVED** [review]
Hey, @ksigei Good work done 
No linter errors 
Correct git workflow
I approve of your project.

File: models/author.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    @first_name = first_name
+    @last_name = last_name
+    @id = Random.rand(1..1000)
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.author = self
+  end
```
Comment (RitobrotoMukherjee) [inline models/author.rb:15]: - This function is wrong, as `Item class` does not have any `author` method. The inherited method name is `add_author`.
- Without implementing this change you will fail during **unit tests**
- Your `add_item` method can be implemented as below:
```
 def add_item(item)
    @items << item
    item.add_author(self)
  end
```

**Review by RitobrotoMukherjee — CHANGES_REQUESTED** [review]
Hi @ksigei 👋 

Great work implementing the project requirements. ✅ 
But there are still a few changes needed to go to the next project. 🧑‍💼 Please check the below comments. 🔬  

## Key Highlights
- Good PR 👍 
- Good commit messages 💯 
- Nice code structure 🥇
---
so-close-2

File: models/author.rb (hunk 1 above)
Comment (ksigei) [reply models/author.rb]: @RitobrotoMukherjee thanks for the correction, I have solved

**Pull request merged**